script.groovy

```diff
@@ -1,10 +1,8 @@
 def getGitChanges() {
-    if(currentBuild.changeSets.size() > 0) {
-        return true
-    }
-    else {
-        return false
-    }
-}
+    if(currentBuild.changeSets.size() > 0)
+      return true;
+    else
+      return false;
```

Comment:
i'm not familiar with the syntax, but can this be simplified to `return currentBuild.changeSets.size() > 0`